### PR TITLE
Update assertiveness.md

### PR DESCRIPTION
**Pull request by relasine**

### CODE REVIEW

**Comment by relasine** [comment]
@emhickmann

File: m4_sessions/1811_inning/assertiveness.md [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
 
-
-
-## Name of Session: 
+## Name of Session: Developing Healthy Assertiveness in the Workplace
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+Do you have trouble voicing your opinion at crucial moments? Ever wanted a raise, but didn't know how to ask? Don't know how to ask your co-worker to turn the music down? Come to the Healthy Assertiveness Gear-Up.
```
Comment (emhickmann) [inline m4_sessions/1811_inning/assertiveness.md:10]: This is a great description.

File: m4_sessions/1811_inning/assertiveness.md [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
-     # Intro to new material
-     # Small Group Activity
-     # Closing Discussion
+* What does "assertive" mean to you?
+* What are good practices for being assertive?
+* What are bad practices for being assertive?
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+* Intro - why did you select this session, on a scale of 1-10, how assertive do you think you are?
```
Comment (emhickmann) [inline m4_sessions/1811_inning/assertiveness.md:25]: Love this! Maybe even have them line up in the room...

File: m4_sessions/1811_inning/assertiveness.md [hunk 3]
```diff
@@ -1,30 +1,36 @@
...
+* Journal - when did your assertiveness (or lack of) play a role in an outcome?
```
Comment (emhickmann) [inline m4_sessions/1811_inning/assertiveness.md:26]: This is a great starter!

File: m4_sessions/1811_inning/assertiveness.md [hunk 4]
```diff
@@ -1,30 +1,36 @@
...
+* Share journals by volunteer
+* Warm-up - physical/vocal
```
Comment (emhickmann) [inline m4_sessions/1811_inning/assertiveness.md:28]: Make sure to narrow this down next planning session

File: m4_sessions/1811_inning/assertiveness.md [hunk 5]
```diff
@@ -1,30 +1,36 @@
...
+* Leaders role-play a salary discussion with feedback
```
Comment (emhickmann) [inline m4_sessions/1811_inning/assertiveness.md:29]: Maybe before this, do you have a few resources on ways to be "assertive" or how to reflect on the level of assertiveness needed for a person?

File: m4_sessions/1811_inning/assertiveness.md [hunk 6]
```diff
@@ -1,30 +1,36 @@
...
+* Small group-based role-play (low-stakes)
```
Comment (emhickmann) [inline m4_sessions/1811_inning/assertiveness.md:30]: This is great.  Please add these to the session so other groups could use this in the future!

File: m4_sessions/1811_inning/assertiveness.md [hunk 7]
```diff
@@ -1,30 +1,36 @@
...
+* Volunteers for demonstrate advanced assertiveness
+* Reflection journaling
```
Comment (emhickmann) [inline m4_sessions/1811_inning/assertiveness.md:32]: What is the final reflection?  How can someone continue to improve their assertiveness?

**Review by emhickmann — COMMENTED** [review]
@relasine Great start to this session.  Review my comments and use the last planning session to get even more specific!

**Pull request merged**